phoenicis-library/src/main/java/org/phoenicis/library/LibraryManager.java

```diff
@@ -80,8 +80,12 @@ public LibraryManager(String shortcutDirectory, ObjectMapper objectMapper) {
         List<ShortcutCategoryDTO> shortcuts = new ArrayList<>();
         for (Map.Entry<String, List<ShortcutDTO>> entry : categoryMap.entrySet()) {
             entry.getValue().sort(ShortcutDTO.nameComparator());
-            ShortcutCategoryDTO category = new ShortcutCategoryDTO.Builder().withId(entry.getKey())
-                    .withName(entry.getKey()).withShortcuts(entry.getValue()).build();
+            ShortcutCategoryDTO category = new ShortcutCategoryDTO.Builder()
+                    .withId(entry.getKey())
+                    .withName(entry.getKey())
+                    .withShortcuts(entry.getValue())
+                    .withIcon(entry.getValue().get(0).getCategoryIcon()) // choose one category icon
```

Comment:
Is it possible that the shortcuts list is empty?

Reply:
No. The entry in the `categoryMap` is created only if there is at least one `ShortcutDTO`.